enclave/src/main/java/tech/pegasys/pantheon/enclave/Enclave.java

```diff
@@ -68,6 +69,10 @@ public ReceiveResponse receive(final ReceiveRequest content) throws Exception {
     return executePost(buildPostRequest(ORION, content, "/receive"), ReceiveResponse.class);
   }
 
+  public boolean pushToHistory(final PushToHistoryRequest content) throws Exception {
+    return executePost(buildPostRequest(JSON, content, "/pushToHistory"), Boolean.class);
```

Comment:
This is not available yet in Orion ...

Reply:
https://github.com/PegaSysEng/orion/pull/283

Reply:
this is also why CI is currently failing - which is a good thing so we cannot prematurely merge this :)